modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/exportimport/data/handler/BatchEnginePortletDataHandler.java

```diff
@@ -212,6 +216,21 @@ private byte[] _getBytes(String fileName, InputStream inputStream)
 		return unsyncByteArrayOutputStream.toByteArray();
 	}
 
+	private String _getImportCreatorStrategy(
+		PortletDataContext portletDataContext) {
+
+		if (UserIdStrategy.CURRENT_USER_ID.equals(
+				MapUtil.getString(
+					portletDataContext.getParameterMap(),
+					PortletDataHandlerKeys.USER_ID_STRATEGY))) {
+
+			return BatchEngineImportTaskConstants.
+				IMPORT_CREATOR_STRATEGY_KEEP_CREATOR;
+		}
+
+		return StringPool.BLANK;
```

Comment:
The current behaviour is to return null to avoid inserting the element into the Map